app/script/event/EventRepository.coffee

```
@@ -329,15 +330,17 @@ class z.event.EventRepository
   ###
   Handle a single event from the notification stream or WebSocket.
   @param event [JSON] Backend event extracted from notification stream
-  @return [Promise] Promise that resolves with boolean whether the event was saved
+  @return [Promise] Resolves with the saved record or boolean true if the event was skipped
   ###
   _handle_event: (event, source) ->
     return new Promise (resolve, reject) =>
       sending_client = event.data?.sender
       if sending_client
         log_message = "Received encrypted event '#{event.type}' from client '#{sending_client}' of user '#{event.from}'"
       else if event.from
-        log_message = "Received unencrypted event '#{event.type}' from user '#{event.from}'"
+        log_message = "Received plain event '#{event.id}' of type '#{event.type}' from client '#{sending_client}' of user '#{event.from}'"
+        if event.type is z.event.Backend.CONVERSATION.MESSAGE_ADD
```

Aren't there more event types that should throw the OUTDATED_SCHEMA error?